src/java/htsjdk/tribble/gelitext/GeliTextCodec.java

```diff
@@ -74,7 +75,7 @@ public GeliTextFeature decode(final String line) {
 
     @Override
     public boolean canDecode(String path){
-	return path.toLowerCase().endsWith(".geli.calls") || path.toLowerCase().endsWith(".geli");
+	return ParsingUtils.pathHasExtension(path, ".geli.calls");
```

Comment:
Don't the `canDecode()` methods for most of the other codecs need an equivalent change? (eg., `BEDCodec.canDecode()`, etc.)